### PR TITLE
19 created home controller and sample landing page

### LINKED ISSUE #19: Landing page: creating controller and view

### PR DESCRIPTION
**Pull request by borvux**
close #19

### CODE REVIEW

File: app/controllers/home_controller.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class HomeController < ApplicationController
+	def index
```
Comment (rezahusain) [inline app/controllers/home_controller.rb:2]: Definitely fix the indenting here

**Review by rezahusain — APPROVED** [review]
Looks like a good start! Keep it up Benny :)

**Review by MayaS1111 — APPROVED** [review]
After you fix the indenting you are all good! Also, make sure to have descriptions for your pull requests; even if they are short.

**Review by APatel-AI — APPROVED** [review]
Looks good

**Pull request merged**